xpcom/ds/nsHashtablesFwd.h

```diff
@@ -91,4 +94,9 @@ template <class KeyType>
 using nsTHashSet =
     nsTBaseHashSet<typename mozilla::detail::nsKeyClass<KeyType>::type>;
 
+template <class KeyType>
+using nsTReplayStableHashSet =
+    nsTBaseHashSet<
+      typename mozilla::detail::nsReplayStableKeyClass<KeyType>::type>;
```

Comment:
Does the formatter split lines 99 and 100 or is this a manual choice?

Reply:
No formatter for gecko code, it's a manual choice.